shared/src/Datadog.Trace.ClrProfiler.Native/CMakeLists.txt

```diff
@@ -102,14 +108,22 @@ SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${OUTPUT_BIN_DIR})
 # ******************************************************
 
 # Sets compiler options
-add_compile_options(-std=c++20 -fPIC -fms-extensions -fvisibility=hidden -g)
+add_compile_options(-std=c++20 -fPIC -fms-extensions -g)
 add_compile_options(-DPAL_STDCPP_COMPAT -DPLATFORM_UNIX -DUNICODE)
 add_compile_options(-Wno-invalid-noreturn -Wno-macro-redefined)
-if (ISMACOS)
-    add_compile_options(-stdlib=libc++ -DMACOS -Wno-pragma-pack -Wno-deprecated-declarations)
-elseif(ISLINUX)
-    add_compile_options(-stdlib=libstdc++ -DLINUX -Wno-pragmas)
+
+if (UNIVERSAL)
+    add_compile_options(-stdlib=libc++ -DUNIVERSAL)
```

Comment:
Shouldn't the value `ON`/`OFF` should also be assigned? 🤔

Reply:
a CMake option is merely a boolean. So we can use it like that